packages/api/database/seeds/06_PermissionSeeder.js

```diff
@@ -218,7 +220,12 @@ class PermissionSeeder {
 		await adminRole
 			.permissions()
 			.attach(
-				[serviceOrderPermissions[1]?.id || null, ...adminPermissionsIds].filter(Boolean),
+				[
+					serviceOrderPermissions[1]?.id || null,
+					serviceOrderPermissions[2]?.id || null,
+					serviceOrderPermissions[3]?.id || null,
```

Comment:
Já deve ser filtrado pelo `filter(Boolean)` caso não exista pois seu valor será `undefined` :)
```suggestion
					serviceOrderPermissions[1]?.id,
					serviceOrderPermissions[2]?.id,
					serviceOrderPermissions[3]?.id,
```